modules/installation-launching-installer.adoc

```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
 The installation program connects to your vCenter instance.
 .. Select the datacenter in your vCenter instance to connect to.
 .. Select the default vCenter datastore to use.
++
+[NOTE]
+====
+DataStore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking DataStore and cluster names.
```

Comment:
@Wiharris the "s" in Datastore is not capitalized in vSphere's documentation. Example usage: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-3CC7078E-9C30-402C-B2E1-2542BEE67E8F.html

Reply:
Done! :)